suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx

```diff
@@ -44,7 +44,6 @@ type PassphraseFormProps = {
 
 const formStyle = prepareNativeStyle(utils => ({
     backgroundColor: utils.colors.backgroundSurfaceElevation1,
-    borderRadius: utils.borders.radii.r24,
```

Comment:
There doesn't seem to be any visible border so it doesn't make sense here. Correct me if I'm wrong. 🙏

Reply:
You don't need border to make something rounded, background is enough

Reply:
I understand that, I'm saying there's no visible effect of the radius change. I believe event the background colour is redundant here.

Reply:
seems like that

Reply:
Removed the form style altogether. 1412384